### PR TITLE
DT-46: Disable DAA work until we can get back to it in the future.

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DT-46

### Summary
DAA work has been pushed off for at least the next six months. In the meantime, we should disable it so that dev work is closer to what is released to production.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rjohanek — APPROVED** [review]
Does this have any testing repercussions? Are there tests we need to disable or add or anything like that?

**Comment by rushtong** [comment]
> Does this have any testing repercussions? Are there tests we need to disable or add or anything like that?

Our tests are 100% component tests and all tests passed, so there should be no issues on that front.

File: src/utils/DAAUtils.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import {checkEnv, envGroups} from './EnvironmentUtils';
-
 export const DAAUtils = {
   isEnabled: (): boolean => {
-    return checkEnv(envGroups.NON_PROD);
+    // TODO: When we get back to working on, this, enable with `checkEnv(envGroups.NON_PROD);`
```
Comment (pshapiro4broad) [inline src/utils/DAAUtils.ts]: typo?
```suggestion
    // TODO: When we get back to working on this, enable with `checkEnv(envGroups.NON_PROD);`
```

Also, IMO it's fine to not have a TODO here as this information is available via git history too

**Review by pshapiro4broad — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]
thank you, this will help me test potential dataset issues in the Data Library 👍

**Pull request merged**